h2o-admissibleml/src/main/java/hex/schemas/InfogramV3.java

```diff
@@ -0,0 +1,278 @@
+     */
+    @API(help = "Maximum relative size of the training data after balancing class counts (can be less than 1.0). " +
+            "Requires balance_classes.", /* dmin=1e-3, */ level = API.Level.expert, direction = API.Direction.INOUT)
+    public float max_after_balance_size;
+
+    @API(help = "Machine learning algorithm chosen to build the infogram.  AUTO default to GBM.", values={"AUTO",
+            "deeplearning", "drf", "gbm", "glm", "xgboost"}, level = API.Level.expert, 
+            direction = API.Direction.INOUT, gridable=true)
+    public InfogramModel.InfogramParameters.Algorithm algorithm;
+
+    @API(help = "Parameters specified to the chosen algorithm can be passed to infogram using algorithm_params.",
+            level = API.Level.expert, gridable=true)
+    public String algorithm_params;
+
+    @API(help = "Predictors that are to be excluded from model due to them being discriminatory or inappropriate for" +
+            " whatever reason.", level = API.Level.secondary, gridable=true)
+    public String[] protected_columns;
+
+    @API(help = "Conditional information for core infogram threshold between 0 and 1 that is used to decide whether a " +
+            "predictor's conditional information is high enough to be chosen into the admissible feature set.  " +
+            "Default to -1 which will be set to 0.1 eventually.",
+            level = API.Level.secondary, gridable = true)
+    public double net_information_threshold;
+
+    @API(help = "Conditional information for fair infogram threshold between 0 and 1 that is used to decide whether a" +
+            " predictor's conditional information is high enough to be chosen into the admissible feature set.  " +
+            "Default to -1 which will be set to 0.1 eventually.",
+            level = API.Level.secondary, gridable = true)
+    public double safety_index_threshold;
+
+    @API(help = "Relevance threshold for fair infogram between 0 and 1 that is used to decide whether a predictor's" +
+            " relevance level is high enough to be chosen into the admissible feature set.  Default to -1 which will" +
+            " be set to 0.1 eventually.", 
+            level = API.Level.secondary, gridable = true)
+    public double relevance_index_threshold;
+
+    @API(help = "Relevance threshold for core infogram between 0 and 1 that is used to decide whether a predictor's" +
+            " relevance level is high enough to be chosen into the admissible feature set.  Default to -1 which will" +
+            " be set to 0.1 eventually.",
+            level = API.Level.secondary, gridable = true)
+    public double total_information_threshold;
+
+    @API(help = "Fraction of training frame to use to build the infogram model.  Default to 1.0",
+            level = API.Level.secondary, gridable = true)
+    public double data_fraction;
+
+    @API(help = "Number of top k variables to consider based on the varimp.  Default to 0.0 which is to consider" +
```

Comment:
```suggestion
    @API(help = "Number of top n variables to consider based on the variable importance.  Defaults to 0.0 which is to consider" +
```

Reply:
Need to keep the k from topk variable name

Reply:
See previous comments.